_opensearch/rest-api/snapshots/get-snapshot-repository

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: Get Snapshot Repository
+parent: Snapshot APIs
+grand_parent: REST API reference
+nav_order: 3
+---
+
+## Get snapshot repository.
+
+Retrieves information about a snapshot repository.
+
+To learn more about repositories, see [Register repository]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#register-repository).
+
+### Path parameters
+
+| Parameter | Data Type | Description |
+| :--- | :--- | :--- |
+| repository | String | A comma-separated list of snapshot repository names to retrieve. Wildcard (`*`) expressions are supported including combining wildcards with exclude patterns starting with `-`. |
+
+### Query parameters
+
+| Parameter | Data Type | Description | 
+:--- | :--- | :---
+| local | Boolean | Whether to get information from the local node. Optional, defaults to `false`.|
+| master_timeout | Time | Amount of time to wait for a connection to the master node. Optiona, defaults to 30 seconds. |
+
+#### Sample request
+
+The following request retrieves information for the `my-opensearch-repo` repository:
+
+````json
+GET /_snapshot/my-opensearch-repo
+````
+
+#### Sample response
+
+Upon success, the response returns repositry information. This sample is for an `s3` repository type.
```

Comment:
I would provide examples for both s3 and fs.

Reply:
Thank you. I was not able to do the configuration needed to create fs type repositories, so I can't add a sample. Nate said we would address the issue at some point in the future.